postgresql/config/functions.sh

```diff
@@ -58,14 +62,13 @@ master_ready() {
 
 bootstrap_replica_via_pg_basebackup() {
   echo 'Bootstrapping replica via pg_basebackup from leader '
-
-  rm -rf {{pkg.svc_data_path}}/*
-  pg_basebackup --pgdata={{pkg.svc_data_path}} --xlog-method=stream --dbname='postgres://{{cfg.replication.name}}@{{svc.leader.sys.ip}}:{{cfg.port}}/postgres'
+  rm -rf {{pkg.svc_data_path}}/pgdata/*
+  pg_basebackup --pgdata={{pkg.svc_data_path}}/pgdata --xlog-method=stream --dbname='postgres://{{cfg.replication.name}}@{{svc.leader.sys.ip}}:{{cfg.port}}/postgres'
```

Comment:
Not blocking: a basebackup can take a long time for real-world databases.  `-v -P` will give you some more output that can be reassuring when you are on minute 5-6 (or 20-21) of bootstrapping :D.

Reply:
great idea!  done